spec/actors/hyrax/actors/file_actor_spec.rb

```diff
@@ -143,7 +142,7 @@ class FileSetWithExtras < FileSet
     let(:query_service) { Wings::Valkyrie::QueryService.new(adapter: metadata_adapter) }
     let(:file_node) do
       node_builder = Wings::FileNodeBuilder.new(storage_adapter: storage_adapter, persister: persister)
-      node = Wings::FileNode.for(file: fixture)
+      node = Hyrax::FileNode.for(file: fixture)
```

Comment:
All changes in this file are related to the namespace change moving FileNode to Hyrax.